### ISSUE TITLE
DIY Labware

### ISSUE THREAD

**Issue by wumbor** [MozillaFestival/open-leaders-7#81]
Project Lead: @wumbor 

Mentor: @GrantRVD

Welcome to OL7, Cohort D! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***

Before Week 1 (Jan 30): Your first mentorship call
- [x] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Feb 6): First Cohort Call (Open by Design)
- [x] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [x] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [x] Check the [Syllabus](https://mzl.la/ol7-syllabus) for notes and connection info for all the cohort calls.

Before Week 3 (Feb 13): Mentorship call
- [x] Look up two other projects and comment on their issues with feedback on their vision statement.
- [x] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [x] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Feb 20): Cohort Call (Build for Understanding)
- [x] Look up two other projects and comment on their issues with feedback on their open canvas.
- [x] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [ ] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.

Week 5 and more
- [ ] Create a repository for your project. New to GitHub? You can [fork our templates](https://mozilla.github.io/global-sprint/project-lead-guide/templates/).
- [ ] Add [Contributing Guidlines](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/)
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/)
- [ ] [Submit your project to Mozilla Pulse](https://www.mozillapulse.org/add), add the tag `#mozsprint`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Leaders. Please refer to the [OL7 Syllabus](https://mzl.la/ol7-syllabus) for more detailed weekly notes and assignments past week 4.

**Comment by wumbor** [comment]
Vision and Mission Statement:
 I am working with faculty in universities to develop open source lab equipment so they can conduct better research and provide practical training to their students. During the open leaders program, I want to develop a GUI control interface and hardware upgrades for my first project [Actifield](https://github.com/trendinafrica/actifield), a simple automated actimeter. I will appreciate any help with this.

**Comment by SanliFaez** [comment]
I am very interested in learning more about your project. Does is also include connecting to other similar projects and resources?

**Comment by annefou** [comment]
That is very interesting! I am definitely interested and will keep an eye; I hope to be able to contribute too.

**Comment by wumbor** [comment]
> I am very interested in learning more about your project. Does is also include connecting to other similar projects and resources?

Yes. My first goal is to create the awareness about open science hardware so I will be collating projects and resources that academics can quickly look up.

**Comment by jmtaylor86** [comment]
> Vision and Mission Statement:
> I am working with faculty in universities to develop open source lab equipment so they can conduct better research and provide practical training to their students. During the open leaders program, I want to develop a GUI control interface and hardware upgrades for my first project [Actifield](https://github.com/trendinafrica/actifield), a simple automated actimeter. I will appreciate any help with this.

Very interesting and such a cool approach. I look forward to learning more!

**Comment by wumbor** [comment]
My Open Canvas https://docs.google.com/presentation/d/1SyGH7A97cMilYWeriBJSFUMbyOzXMQ4vTwrjxvWdBmY/edit#slide=id.p

**Comment by lsrkthelibrarian** [comment]
This sounds great, but as a non scientist, I wonder if you can elaborate a little bit more on "open source lab equipment."  Often, when I think of open source, I think software, but I am curious to know if you are referring to hardware?  As a follow up to another inquiry, you did say that you will be creating awareness of the hardware options.  Can you clarify with some examples of open source hardware?  Thanks!!

**Comment by wumbor** [comment]
My roadmap. Still in the very early stages https://docs.google.com/document/d/1J3hzlSDpMEGiaYzxsciNKDaoqXcfJCV8WQiXeuSdRT0/edit?usp=sharing

**Comment by wumbor** [comment]
> This sounds great, but as a non scientist, I wonder if you can elaborate a little bit more on "open source lab equipment." Often, when I think of open source, I think software, but I am curious to know if you are referring to hardware? As a follow up to another inquiry, you did say that you will be creating awareness of the hardware options. Can you clarify with some examples of open source hardware? Thanks!!

**Comment by wumbor** [comment]
> This sounds great, but as a non scientist, I wonder if you can elaborate a little bit more on "open source lab equipment." Often, when I think of open source, I think software, but I am curious to know if you are referring to hardware? As a follow up to another inquiry, you did say that you will be creating awareness of the hardware options. Can you clarify with some examples of open source hardware? Thanks!!

Hello @lsrkthelibrarian . By "open source lab equipment," I mean Open Science Hardware. This basically refers to "any piece of hardware used for scientific investigations that can be obtained, assembled, used, studied, modified, shared, and sold by anyone." (borrowing the definition by [GOSH ](http://openhardware.science/)). So similar to open software, the open hardware makes all the designs, code and necessary instructions for anyone else to replicate the equipment freely available.

[Open Behaviour](http://openbehavior.com/) and [Open Labware ](https://open-labware.net/) host a collection of open hardware projects.

**Comment by jurra** [comment]
Great Idea!!
